### PR TITLE
feat/171: Create-career-page 

### LINKED ISSUE #171: [Feature]:  Create Careers Page
### Description

Create and Implement the Careers Page. This page must be fully responsive and implemented according to the design. The page should fully utilize any reusable components

### Acceptance Criteria

Page must be fully responsive on mobile, tablets and desktops
Page must fully implement what was designed
Page must use reusable components already existing

### Links

https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=10-23157&m=dev

### Image
Screenshot 2024-07-18 at 21 09 11

### Related Issues
This page needs these reusable components, Please follow up
[#issue115](https://github.com/hngprojects/hng_boilerplate_nextjs/issues/115),
[#issue119](https://github.com/hngprojects/hng_boilerplate_nextjs/issues/119),
[#issue110](https://github.com/hngprojects/hng_boilerplate_nextjs/issues/110)

### PR DESCRIPTION
**Pull request by Pharm-ack**
# Description

**Closes #171 **

# Changes proposed

## What were you told to do?
I replicated and styled the career page component to match the figma design
I created the route that lead to the career page
I ensured the design was responsive on larger screens, tablets and mobile based on the design
FIGMA LINK
https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=10-23157&m=dev

## What did you do?

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/Videos
Annotation 2024-07-21 031437

screenbud-5be71982-246b-43e8-94a0-96a8bc824938

screenbud-37b861c8-b919-4dff-a168-4817542576c1

Annotation 2024-07-21 033158

screenbud-c774ee57-9805-4eb3-91c5-efad6fc2d1fe

### CODE REVIEW

**Comment by oreoluwa212** [comment]
Confusing my life

**Comment by Pharm-ack** [comment]
My laptop screen is small that why I couldn't capture every page at once.

File: src/app/career/page.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import CareerCardParent from "../../components/common/CareerCard";
```
Comment (Justfemi) [inline src/app/career/page.tsx]: Use ~ instead of relative paths

**Review by Justfemi — CHANGES_REQUESTED** [review]

File: src/app/career/page.tsx (hunk 1 above)
Comment (Pharm-ack) [reply src/app/career/page.tsx]: I have fix it now

**Review by Justfemi — APPROVED** [review]

**Comment by Pharm-ack** [comment]
Fix the layout

**Pull request merged**